### PR TITLE
feat: add custom copilot basic template

### PR DESCRIPTION
**Pull request by kuojianlu**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/26356837

### CODE REVIEW

File: templates/js/custom-copilot-basic/README.md.tpl [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Overview of the AI Chat Bot template
```
Comment (SLdragon) [inline templates/js/custom-copilot-basic/README.md.tpl:1]: Each template may need different README, do we really need this in basic template?

File: templates/js/custom-copilot-basic/src/app/app.js.tpl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+// Create AI components
+const model = new OpenAIModel({
+  {{#useOpenAI}}
+  apiKey: config.openAIKey,
+  defaultModel: "gpt-3.5-turbo",
+  {{/useOpenAI}}
+  {{#useAzureOpenAI}}
+  azureApiKey: config.azureOpenAIKey,
+  azureDefaultDeployment: "gpt-35-turbo",
```
Comment (SLdragon) [inline templates/js/custom-copilot-basic/src/app/app.js.tpl:16]: How about add a comment to tell user don't forget to  update it himself

File: templates/ts/custom-copilot-basic/src/prompts/chat/skprompt.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+You are an AI bot that can chat with users.
```
Comment (KennethBWSong) [inline templates/ts/custom-copilot-basic/src/prompts/chat/skprompt.txt:1]: [Minor] @kuojianlu We are going to reuse this template for Custom API template, and we need to update this file. We will overwrite config.json and skprompt for update/add some description from OpenAPI spec. Please let me know if you have any better method for updating these files.

File: templates/js/custom-copilot-basic/README.md.tpl (hunk 1 above)
Comment (kuojianlu) [reply templates/js/custom-copilot-basic/README.md.tpl:1]: This basic template is the whole template, not the common part that will be reused by other templates.
image

File: templates/js/custom-copilot-basic/src/app/app.js.tpl (hunk 2 above)
Comment (kuojianlu) [reply templates/js/custom-copilot-basic/src/app/app.js.tpl:16]: The hint is included in readme file.

File: templates/js/custom-copilot-basic/README.md.tpl [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
```
Comment (MuyangAmigo) [inline templates/js/custom-copilot-basic/README.md.tpl:2]: Title: `Overview of the Basic AI Chat Bot template`, the title needs to align with what's shown in the UX

**Review by MuyangAmigo — CHANGES_REQUESTED** [review]

File: templates/js/custom-copilot-basic/README.md.tpl (hunk 4 above)
Comment (kuojianlu) [reply templates/js/custom-copilot-basic/README.md.tpl:2]: Good point. Updated.

**Review by MuyangAmigo — APPROVED** [review]

File: templates/js/custom-copilot-basic/.vscode/launch.json.tpl [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+{{/enableTestToolByDefault}}
+{{^enableTestToolByDefault}}
+                "group": "1-local",
+{{/enableTestToolByDefault}}
+                "order": 2
+            },
+            "stopAll": true
+        },
+        {
+            "name": "Debug in Test Tool (Preview)",
```
Comment (kimizhu) [inline templates/js/custom-copilot-basic/.vscode/launch.json.tpl:105]: Let’s talk to Qinen / Aocheng if preview label can be removed this time.

File: templates/js/custom-copilot-basic/README.md.tpl [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+{{/useAzureOpenAI}}
+1. Press F5 to start debugging which launches your app in Teams App Test Tool using a web browser. Select `Debug in Test Tool (Preview)`.
+1. You can send any message to get a response from the bot.
+
+**Congratulations**! You are running an application that can now interact with users in Teams App Test Tool:
+
+![ai chat bot](https://github.com/OfficeDev/TeamsFx/assets/9698542/9bd22201-8fda-4252-a0b3-79531c963e5e)
+{{/enableTestToolByDefault}}
+{{^enableTestToolByDefault}}
+1. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
```
Comment (kimizhu) [inline templates/js/custom-copilot-basic/README.md.tpl:48]: if you haven’t already -> if you haven’t yet?

Comment (kuojianlu) [reply templates/js/custom-copilot-basic/README.md.tpl]: Updated.

**Review by kimizhu — APPROVED** [review]

**Pull request merged**